### PR TITLE
Adding my hometown file

### LINKED ISSUE #6: aruddell Hometown
## Portadown, Nothern Ireland
- [x] Create branch
- [x] Add my file
- [x] Commit my file
- [x] Open pull request
- [ ] Get :+1: 
- [ ] Merge pull request

### PR DESCRIPTION
**Pull request by aruddell**
This Pull request closes #6

### CODE REVIEW

File: places/Portadown.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### Portadown 
+
+- County Armagh
+- Around 30,000 people
```
Comment (aruddell) [inline places/Portadown.md:4]: Not exact figure

Comment (k-vaughan) [inline places/Portadown.md:4]: Cite source please.

Comment (MichaelReel) [inline places/Portadown.md:4]: Any trains stop there? @aruddell

Comment (aruddell) [inline places/Portadown.md:4]: Not the last few days because Lurgan people putting suspicious devices everywhere @MichaelReel

Comment (aruddell) [inline places/Portadown.md:4]: I counted them all once @k-vaughan

**Comment by MichaelReel** [comment]
:+1: :sunglasses: :ship:

**Pull request merged**